DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/ManageCourseController.cs

```diff
@@ -99,5 +114,98 @@ EditLearningPathwayDefaultsViewModel model
 
             return RedirectToAction("Index", new { customisationId = model.CustomisationId });
         }
+
+        [HttpGet]
+        [ServiceFilter(typeof(RedirectEmptySessionData<EditLearningPathwayDefaultsData>))]
+        [Route("AutoRefreshOptions")]
+        public IActionResult EditAutoRefreshOptions(int customisationId)
+        {
+            var centreId = User.GetCentreId();
+            var categoryId = User.GetAdminCategoryId()!;
+
+            var courseDetails = courseService.GetCourseDetailsForAdminCategoryId(
+                customisationId,
+                centreId,
+                categoryId.Value
+            );
+
+            var model = new EditAutoRefreshOptionsViewModel(courseDetails!);
+
+            SetViewBagCourseOptions(customisationId, model.RefreshToCustomisationId);
+
+            return View(model);
+        }
+
+        [HttpPost]
+        [ServiceFilter(typeof(RedirectEmptySessionData<EditLearningPathwayDefaultsData>))]
+        [Route("AutoRefreshOptions")]
+        public IActionResult SaveAutoRefreshOptions(

```

Comment:
I'd call this the same as the Get endpoint. This means you won't have to specify view name when returning when the ModelState is invalid.